### ISSUE TITLE
fortios_system_interface runs into errors with 2.1.1

### ISSUE THREAD

**Issue by tregad** [fortinet-ansible-dev/ansible-galaxy-fortios-collection#119]
We're running into this error with version 2.1.1 of the modules. Worked with 2.1.0:
```
An exception occurred during task execution. To see the full traceback,
use -vvv. The error was: TypeError: 'NoneType' object is not iterable
failed: [prdfw01.int.15knetworks.com] (item={'vdom': 'CUSTOM', 'state': 'present',
'name': 'VLAN_1114', 'allowaccess': 'ping', 'alias':
'GLOBAL_LINUX_RZ_SERVICES', 'device_identification': 'enable',
'interface': 'LACP_CUSTOM', 'ip': '10.11.14.1 255.255.255.0', 'role':
'lan', 'vlanid': 1114}) => {"ansible_loop_var": "item", "changed":
false, "item": {"alias": "GLOBAL_LINUX_RZ_SERVICES", "allowaccess":
"ping", "device_identification": "enable", "interface": "LACP_CUSTOM",
"ip": "10.11.14.1 255.255.255.0", "name": "VLAN_1114", "role": "lan",
"state": "present", "vdom": "CUSTOM", "vlanid": 1114}, "module_stderr":
"Traceback (most recent call last):\n  File
\"/home/mv.com/gerth.adrian/.ansible/tmp/ansible-local-
12067764afloxob/ansible-tmp-1625133973.0677059-1206818-
129884104181545/AnsiballZ_fortios_system_interface.py\", line 102, in
<module>\n    _ansiballz_main()\n  File
\"/home/mv.com/gerth.adrian/.ansible/tmp/ansible-local-
12067764afloxob/ansible-tmp-1625133973.0677059-1206818-
129884104181545/AnsiballZ_fortios_system_interface.py\", line 94, in
_ansiballz_main\n    invoke_module(zipped_mod, temp_path,
ANSIBALLZ_PARAMS)\n  File
\"/home/mv.com/gerth.adrian/.ansible/tmp/ansible-local-
12067764afloxob/ansible-tmp-1625133973.0677059-1206818-
129884104181545/AnsiballZ_fortios_system_interface.py\", line 40, in
invoke_module\n   
runpy.run_module(mod_name='ansible_collections.fortinet.fortios.plugins
.modules.fortios_system_interface', init_globals=None,
run_name='__main__', alter_sys=True)\n  File
\"/usr/lib64/python3.6/runpy.py\", line 205, in run_module\n    return
_run_module_code(code, init_globals, run_name, mod_spec)\n  File
\"/usr/lib64/python3.6/runpy.py\", line 96, in _run_module_code\n   
mod_name, mod_spec, pkg_name, script_name)\n  File
\"/usr/lib64/python3.6/runpy.py\", line 85, in _run_code\n   
exec(code, run_globals)\n  File
\"/tmp/ansible_fortios_system_interface_payload_tjmpmtls/ansible_fortio
s_system_interface_payload.zip/ansible_collections/fortinet/fortios/plu
gins/modules/fortios_system_interface.py\", line 13060, in <module>\n 
File
\"/tmp/ansible_fortios_system_interface_payload_tjmpmtls/ansible_fortio
s_system_interface_payload.zip/ansible_collections/fortinet/fortios/plu
gins/modules/fortios_system_interface.py\", line 13039, in main\n  File
\"/tmp/ansible_fortios_system_interface_payload_tjmpmtls/ansible_fortio
s_system_interface_payload.zip/ansible_collections/fortinet/fortios/plu
gins/modules/fortios_system_interface.py\", line 2513, in
fortios_system\n  File
\"/tmp/ansible_fortios_system_interface_payload_tjmpmtls/ansible_fortio
s_system_interface_payload.zip/ansible_collections/fortinet/fortios/plu
gins/modules/fortios_system_interface.py\", line 2454, in
system_interface\n  File
\"/tmp/ansible_fortios_system_interface_payload_tjmpmtls/ansible_fortio
s_system_interface_payload.zip/ansible_collections/fortinet/fortios/plu
gins/modules/fortios_system_interface.py\", line 2429, in
flatten_multilists_attributes\n  File
\"/tmp/ansible_fortios_system_interface_payload_tjmpmtls/ansible_fortio
s_system_interface_payload.zip/ansible_collections/fortinet/fortios/plu
gins/modules/fortios_system_interface.py\", line 2416, in
flatten_single_path\nTypeError: 'NoneType' object is not iterable\n",
"module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the
exact error", "rc": 1}
An exception occurred during task execution. To see the full traceback,
use -vvv. The error was: TypeError: 'NoneType' object is not iterable
```
I think the error here is on line 2412

**Comment by tregad** [comment]
#121 added

**Comment by WintyBe** [comment]
I can confirm the above fix works.

**Comment by JieX19** [comment]
Hi @tregad @WintyBe

The error is fixed in the latest version 2.1.2, please install it using **ansible-galaxy collection install fortinet.fortios:2.1.1**

Thanks.
